diff --git a/dist/components/Input/Input.svelte b/dist/components/Input/Input.svelte
index 0689bc651d318c4c894eb0add9992aaf8e668726..f82f9104325e70a39a36cd94d7ab21c279d7e217 100644
--- a/dist/components/Input/Input.svelte
+++ b/dist/components/Input/Input.svelte
@@ -38,7 +38,7 @@
   });
 
   const containerStyles = tv({
-    base: cleanClass(styleVariants.inputContainerCommon, 'mt-0.5 flex w-full items-center'),
+    base: cleanClass(styleVariants.inputContainerCommon, 'flex w-full items-center'),
     variants: {
       shape: styleVariants.shape,
       roundedSize: styleVariants.inputRoundedSize,
diff --git a/dist/styles.js b/dist/styles.js
index 6fb658e9a73ca703004a8ac242bdee8ad40939be..1000349edd8057f83d5c14a33b0a63b1a2b20df1 100644
--- a/dist/styles.js
+++ b/dist/styles.js
@@ -13,7 +13,7 @@ export const styleVariants = {
         muted: 'text-gray-600 dark:text-gray-400',
     },
     inputCommon: 'disabled:bg-gray-300 disabled:text-dark dark:disabled:bg-gray-900 dark:disabled:text-gray-200 bg-transparent transition outline-none disabled:cursor-not-allowed ',
-    inputContainerCommon: 'bg-gray-100 ring-1 ring-gray-200 focus-within:ring-primary dark:focus-within:ring-primary transition outline-none focus-within:ring-1 disabled:cursor-not-allowed dark:bg-gray-800 dark:ring-black',
+    inputContainerCommon: 'bg-gray-100 ring-1 ring-gray-200 focus-within:ring-primary dark:focus-within:ring-primary transition outline-none focus-within:ring-1 disabled:cursor-not-allowed dark:bg-gray-600 dark:ring-black',
     shape: {
         rectangle: 'rounded-none',
         'semi-round': '',
